datastore/longhorn.go

```diff
@@ -2936,7 +2936,7 @@ func (s *DataStore) UpdateInstanceManagerStatus(im *longhorn.InstanceManager) (*
 	return obj, nil
 }
 
-func verifyCreation(name, kind string, getMethod func(name string) (runtime.Object, error)) (runtime.Object, error) {
+func VerifyCreation(name, kind string, getMethod func(name string) (runtime.Object, error)) (runtime.Object, error) {
 	// WORKAROUND: The immedidate read after object's creation can fail.
```

Comment:
Why need to export this function? It seems only used in datastore pkg only.

Reply:
`package backend` will use for verify the creation.

Reply:
But I feel the verification is not necessary as I explained in https://github.com/longhorn/longhorn-manager/pull/1671#issuecomment-1370438033. WDYT? @innobead